sdk/eventhubs/azure-messaging-eventhubs/inc/azure/messaging/eventhubs/models/event_data.hpp

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT License.
 #pragma once
 #include <azure/core/amqp.hpp>
+#include <azure/core/datetime.hpp>
```

Comment:
I am not suggesting to fix in this PR, but it looks like this file could include a whole lot more headers - `vector`, `string`, azure nullable...

It would be great if we could find some tool that could do that and run it across all files in our repo, to add the missing includes and remove the ones that are not necessary.

Reply:
Yes, but if you're a customer and write "`#include <azure/messaging/eventhubs/models/event_data.hpp>`", without including other headers, you get a compilation error.

Reply:
#5092